### ISSUE TITLE
Creating audio-tutorials to help new users on-board to using Soundscape Community

### ISSUE THREAD

**Issue by jchudge** [soundscape-community/soundscape#83]
This is intended to provide an overview of what Soundscape is and how it works; and therefore help the user to get the most from it.

**Comment by jchudge** [comment]
@Mdong9 thanks for looking into this. The initial idea was to provide short audio clips describing specific features of the app...probably in the 30-60 seconds range each. However, if you feel that creating videos would work too, then that's absolutely fine. I'd recommend we still keep it to the same time range to make it easier for people to access and consume the content.

**Comment by Mdong9** [comment]
@jchudge I'll create some sample audio clips and you can see if they fit what you had in mind. Also where would I push the audio clip files if I were to push them, unless you want me to upload them somewhere else for review. (the audio clips will most likely use the .mp3 file format)

**Comment by JJGatchalian** [comment]
If it's possible to create a resource link and have the audio tutorials online rather than having the MP3 files within the app so that the download size wouldn't get too big. We need an internet connection to use Soundscape anyway so why not post it online. There's also a link to the Soundscape Community Youtube channel already in the About page, and you may want to look into that.

**Comment by jchudge** [comment]
@Mdong9  I agree with @JJGatchalian  that linking to them from the app, and storing them on our YouTube channel and website is the best solution for storing them. I'll get that arranged once you have the first few audio clips :-)

**Comment by jchudge** [comment]
@Mdong9 that's great to hear. Can you mail or send link to the files to community.soundscape@gmail.com please

**Comment by Mdong9** [comment]
@jchudge the gmail account should have received a zip file in an email with the subject "Soundscape Audio Guide Files"

**Comment by jchudge** [comment]
@Mdong9  I’ve shared this with our Orientation & Mobility expert at Vision Ireland, and the feedback is great. 
Suggested a small tweak for the Welcome guide so that it reads as follows:

Soundscape “Community” using spatial audio technology to build a richer awareness of your surroundings, enabling you to be more confident and empowered to get around.

So, please do carry on and let me know if you have any questions.
